src/lava/lib/optimization/utils/solver_benchmarker.py

```diff
@@ -0,0 +1,97 @@
+# INTEL CORPORATION CONFIDENTIAL AND PROPRIETARY
```

Comment:
As long as the solver_benchmarker is in the public repo, this license header should be exchanged.